common-kafka-test/src/main/java/com/cerner/common/kafka/testing/KafkaBrokerTestHarness.java

```diff
@@ -145,31 +140,32 @@ public String getClusterId() {
      * Start up the Kafka broker cluster.
      *
      * @throws IOException if an error occurs during Kafka broker startup.
-     * @throws IllegalStateException if the Kafka broker cluster has already been {@link #setUp() setup}.
+     * @throws IllegalStateException if the Kafka broker cluster has already been {@link #setUp(TestInfo) setup}.
      */
     @Override
-    public void setUp() throws IOException {
+    public void setUp(TestInfo testInfo) {
```

Comment:
Add `@param` javadoc for testInfo.

Reply:
Added: https://github.com/cerner/common-kafka/pull/51/commits/cf86c31d346eb58b3a380455d8b1106b4649cb43